contrib/TestHarness2/test_harness/timeout.py

```diff
@@ -54,6 +54,10 @@ def dirs_with_files_matching(path: Path, pattern: Pattern, recurse: bool = True)
                     summary.valgrind_out_file = valgrind_file
                     summary.summarize_files(files)
                     summary.out.dump(sys.stdout)
+            elif config.long_running:
```

Comment:
I don't think this needs to be a separate if block given how similar it is with the normal case, especially if you just use `config.long_running` down the stack instead of passing it as an argument

Reply:
yeah, that makes sense. Done.